src/preferences.ts

```diff
@@ -78,42 +86,47 @@ function recordPersonalDefaults (klass, context) {
           $rdf.st(reg, ns.solid('forClass'), klass, context.preferencesFile)]
       }
       prefs = widgets.newThing(context.preferencesFile)
-      ins.push($rdf.st(reg, ns.solid('personalDefaults'), prefs, context.preferencesFile))
-      kb.updater.update([], ins, function (uri, ok, errm) {
+      ins.push($rdf.st(reg, ns.solid('personalDefaults'), prefs!, context.preferencesFile))
+      kb.updater.update([], ins, function (uri: string, ok: boolean, errm: string) {
         if (!ok) {
           reject(new Error('Setting preferences for ' + klass + ': ' + errm))
         } else {
           context.personalDefaults = prefs
           resolve(context)
         }
       })
-    }, err => {
+    }, (err: Error) => {
       reject(err)
     })
   })
 }
 
-function renderPreferencesForm (subject, klass, preferencesForm, context) {
+function renderPreferencesForm (
+  subject: $rdf.NamedNode,
```

Comment:
Isn't it better with simply `NamedNode`?

Reply:
Haven't heard of that before, why? Should I change the other `$rdf.*` imports as well?